ui/k-toolbar/main.kytos

```diff
@@ -41,6 +56,182 @@
                 }
                 this.$kytos.$emit("showInfoPanel", listWindows)
             },
-        }
+            /*
+                Gets the data fields after button click and consumes POST api to post maintenace window.
+            */
+            post_window(){
+                var self = this
+                this.formatInfo();
+                var request = $.ajax({
+                    async: true,
+                    dataType: "json",
+                    type: "POST",
+                    contentType: "application/json",
+                    data: JSON.stringify({"id": self.id,
+                                          "description": self.description,
+                                          "start": self.start,
+                                          "end": self.end,
+                                          "items": self.merged_list,         
+                    }),
+                    url:this.$kytos_server_api + "kytos/maintenance",
+                });
+
+                request.done(function() {
+                    let notification = {
+                        icon: 'gear',
+                        title: 'Maintenance Window Created',
+                        description: 'Maintenance Window with id ' + self.id + ' was created.'
+                    }
+                    self.$kytos.$emit("setNotification", notification);
+                    // Clearing fields if POST is a success
+                    self.forceRenderer();
+                }),
+                request.fail(function(jqXHR, status, error) {
+
+                    error_message = JSON.parse(jqXHR.responseText)
+                    if (error_message.hasOwnProperty('response')) {
+                        error_message = error_message.response
+                    } else if (error_message.hasOwnProperty('description')) {
+                        error_message = error_message.description
+                    }
+                    if(error_message.includes('%Y-%m-%dT%H:%M:%S%z') || (error_message.includes('start')) || (error_message.includes('past'))){
+                        error_message += '. Remember that the actual format for time is year-month-dayTHour:Minutes:Seconds+0000.'
```

Comment:
@rohanCodes001, the format in the error message was helpful, I wonder if we could make it even more explicit though, something like this:`The expected datetime format is YYYY-MM-DDTHH:MM:SS-0000, for example, 2026-04-22T11:41:01-0000`, what do you think? cc'ing @rmotitsuki too

Reply:
Yes, a real example would be helpful